airbyte-scheduler/client/src/main/java/io/airbyte/scheduler/client/DefaultSchedulerJobClient.java

```diff
@@ -1,91 +0,0 @@
-/*
- * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
- */
-
-package io.airbyte.scheduler.client;
-
-import io.airbyte.config.ActorDefinitionResourceRequirements;
-import io.airbyte.config.DestinationConnection;
-import io.airbyte.config.SourceConnection;
-import io.airbyte.config.StandardSync;
-import io.airbyte.config.StandardSyncOperation;
-import io.airbyte.scheduler.models.Job;
-import io.airbyte.scheduler.persistence.JobCreator;
-import io.airbyte.scheduler.persistence.JobPersistence;
-import java.io.IOException;
-import java.util.List;
-import java.util.Optional;
-import javax.annotation.Nullable;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class DefaultSchedulerJobClient implements SchedulerJobClient {
```

Comment:
This class is no longer needed, since replication jobs are now all submitted through the connection manager temporal workflow